packages/account/src/Helpers/utils.tsx

```diff
@@ -196,9 +196,6 @@ export const isDocumentNumberValid = (document_number: string, document_type: Fo
 
 export const shouldHideHelperImage = (document_id: string) => document_id === IDV_NOT_APPLICABLE_OPTION.id;
 
```

Comment:
while trying it during implementation I moved this function to shared.
later it appeared, that it is not needed currently on this PR, I just decided to keep it in shared to make it reusable